tests/reference_submission_tests.py

```diff
@@ -30,51 +33,132 @@
 FLAGS = flags.FLAGS
 PYTORCH_DEVICE = torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')
 
-_EXPECTED_METRICS = {
-    'mnist': {
-        'jax': {'test/accuracy': 0.0947265625},
-        'pytorch': {'test/accuracy': 0.109375},
-    }
+_EXPECTED_METRIC_NAMES = {
+    'mnist': ['validation/accuracy', 'test/accuracy'],
+    'imagenet': ['validation/accuracy'],
+    'librispeech': ['validation/word_error_rate', 'train/word_error_rate'],
+    'ogbg': [
+        'train/accuracy', 'validation/loss', 'test/mean_average_precision'
+    ],
+    'wmt': ['train/bleu', 'validation/loss', 'test/accuracy'],
 }
```

Comment:
I might be missing something here, but shouldn't we get more than this?
For `mnist`, for example, do we not get the `loss` and `accuracy` for `train`, `valid`, and `test`?

Reply:
yep, I added a few more to better cover the cross product of {metric names, split names}